src/components/relative.js

```diff
@@ -8,21 +8,112 @@ import React, {Component, PropTypes} from 'react';
 import {intlShape, relativeFormatPropTypes} from '../types';
 import {invariantIntlContext, shouldIntlComponentUpdate} from '../utils';
 
+const SECOND = 1000;
+const MINUTE = 1000 * 60;
+const HOUR   = 1000 * 60 * 60;
+const DAY    = 1000 * 60 * 60 * 24;
+
+// The maximum timer delay value is a 32-bit signed integer.
+// See: https://mdn.io/setTimeout
+const MAX_TIMER_DELAY = 2147483647;
+
+function selectUnits(delta) {
+    let absDelta = Math.abs(delta);
```

Comment:
This is my current coding style. I use `const` to represent constant bindings to constant values, generally which are static and not computed.

In places where I'm computing values, like this case, I've been using `let`. I may change my coding style in the future, but I'd want to apply it repo-wide.